src/components/DpCheckbox/DpCheckbox.vue

```diff
@@ -12,6 +12,7 @@
       :checked="checked"
       :value="valueToSend"
       :data-cy="dataCy !== '' ? dataCy : false"
+      :data-dp-validate-error-fieldname="dataDpValidateErrorFieldname || null"
```

Comment:
Why didn't you include the `label.text` mechanism here, too? Because checkbox labels are too long, generally?

I'd suggest to have the following mechanism for all form components, then:

- they all have the prop `dataDpValidateErrorFieldname`, it's value wins
- if not set, but they have label.text set, this one is used
- if no label.text is found (e.g. DpEditor), fallback to generic error

Reply:
thx, done here https://github.com/demos-europe/demosplan-ui/pull/359/commits/aa52da6e7983653787044b4ebc881e065b739532